### PR TITLE
Add Production deployment CI workflow

### PR DESCRIPTION
**Pull request by parth-07**

### CODE REVIEW

File: .github/workflows/production-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    shell: bash
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```
Comment (manorit2001) [inline .github/workflows/production-deploy.yml:19]:
```suggestion
          fetch-depth: 0
          ref: production
```

File: .github/workflows/production-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+      - name: Add heroku production remote
+        run: heroku git:remote --app "${HEROKU_PRODUCTION_APP_NAME}"
+        env:
+          HEROKU_PRODUCTION_APP_NAME: ${{ secrets.HEROKU_PRODUCTION_APP_NAME }}
+
+      - name: Push to heroku master
+        run: |
+          git checkout "${GITHUB_REF##*/}"
+          git push -f heroku "${GITHUB_REF##*/}":master
```
Comment (manorit2001) [inline .github/workflows/production-deploy.yml]:
```suggestion
          git push -f heroku HEAD:master
```

**Review by manorit2001 — APPROVED** [review]

File: .github/workflows/production-deploy.yml (hunk 1 above)
Comment (parth-07) [reply .github/workflows/production-deploy.yml:19]: Good suggestion, modified workflow file.

File: .github/workflows/production-deploy.yml (hunk 2 above)
Comment (parth-07) [reply .github/workflows/production-deploy.yml]: Done.

**Pull request merged**